README.md

```diff
@@ -16,7 +16,34 @@ http://developers.google.com/api-client-library/php
 
 ## Installation ##
 
-For the latest installation and setup instructions, see [the documentation](https://developers.google.com/api-client-library/php/start/installation).
+The [Releases](https://github.com/google/google-api-php-client/releases) page lists all stable versions. Download any file
+with the name `google-api-php-client-[RELEASE_NAME].zip` for a package including this library and its dependencies.
+
+### Composer
+
+Install this library via [composer](https://getcomposer.org) by running the following command:
+
+```sh
+# change directories to the root of your project
+cd /path/to/my-project
+# if you don't already have composer, install it
+curl -sS https://getcomposer.org/installer | php
```

Comment:
Here is the reaosn why.

Well, curling a script from the web and pipe it into an interpreter, might be a trend, but in my opinion, it's dangerous in some senses. When composer website started to suggest more secure way, this document will provide an orphaned, insecure instruction.